### PR TITLE
Reorganize email template files

### PR DESCRIPTION
**Pull request by sgrimm**
The current scheme where all templates are direct children of a single
directory is going to get cluttered. Organize the templates by type, and
extract some common content (links to the app) into include files.

### CODE REVIEW

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #389** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/389" target="_blank">Graphite</a>  👈
    * **PR #390** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/390" target="_blank">Graphite</a> 
      * **PR #391** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/391" target="_blank">Graphite</a> 
        * **PR #392** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/392" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/389?utm_source=stack-comment).

**Comment by karthikbtf** [comment]
I think this review has some changes that will be undone by https://github.com/terraware/terraware-server/pull/390 .. I should have looked at this review first.

**Comment by karthikbtf** [comment]
I think this review has some changes that will be undone by https://github.com/terraware/terraware-server/pull/390 .. I should have looked at this review first.

File: src/main/resources/templates/email/user/userAddedToProject/body.ftlh.mjml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <!-- <#-- @ftlvariable name="" type="com.terraformation.backend.email.model.UserAddedToProject" --> -->
 <mjml>
-    <mj-include path="../head.ftlh.mjml" />
+    <mj-include path="../../head.ftlh.mjml" />
```
Comment (karthikbtf) [inline src/main/resources/templates/email/user/userAddedToProject/body.ftlh.mjml]: I think the directory should be `addedToProject`

File: src/main/resources/templates/email/user/userAddedToOrganization/body.ftlh.mjml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <!-- <#-- @ftlvariable name="" type="com.terraformation.backend.email.model.UserAddedToOrganization" --> -->
 <mjml>
-    <mj-include path="../head.ftlh.mjml" />
+    <mj-include path="../../head.ftlh.mjml" />
```
Comment (karthikbtf) [inline src/main/resources/templates/email/user/userAddedToOrganization/body.ftlh.mjml]: Directory should be `addedToOrganization`

Comment (karthikbtf) [inline src/main/resources/templates/email/user/userAddedToOrganization/body.ftlh.mjml]: Directory should be `addedToOrganization`

**Comment by sgrimm** [comment]
Sorry the sequence here wasn't clear. I guess I didn't do anyone any favors by splitting this up into smaller steps. :(

**Comment by karthikbtf** [comment]
> Sorry the sequence here wasn't clear. I guess I didn't do anyone any favors by splitting this up into smaller steps. :(
It definitely helped @sgrimm , I should have worked my way up. All good.

**Pull request merged**